### PR TITLE
Fix license tag to be an SPDX identifier

### PR DESCRIPTION
**Pull request by nsoranzo**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -54,21 +54,3 @@ extra:
   recipe-maintainers:
     - conda-forge/r
     - nsoranzo
-
```
Comment (bgruening) [inline recipe/meta.yaml:57]: @nsoranzo please let those comments in. We keep them for new recipes as some kind of metadata.

Comment (nsoranzo) [reply recipe/meta.yaml:57]: Many of these metadata will be out-of-date pretty quickly, isn't all these info just a click away?

Comment (bgruening) [reply recipe/meta.yaml:57]: Was not my decision. But I guess ultimately we want also to regenerate the recipes on version update.

Comment (bgruening) [reply recipe/meta.yaml:57]: Either way, its not super important I think.__

**Comment by bgruening** [comment]
Oh .... @nsoranzo what is way more important ... please do never create a branch here. Branches are active, means they are used to keep older versions and we push from branches to anaconda.org.

**Pull request merged**